pallet/Cargo.toml

```diff
@@ -0,0 +1,89 @@
+[package]
+name = "peaq-pallet-rbac"
+version = "0.0.2"
+edition = "2021"
+description = 'A pallet that handles peaq RBAC managment on peaq network.'
+authors = ['peaq <https://github.com/peaqnetwork>']
+homepage = 'https://peaq.network/'
+license = 'Apache-2.0'
+publish = false
+repository = 'https://github.com/peaqnetwork/peaq-pallet-rbac/'
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dev-dependencies]
+hex-literal = '0.3.3'
+
+[dependencies.codec]
+default-features = false
+features = ['derive']
+package = 'parity-scale-codec'
+version = '3.0.0'
+
+[dependencies.frame-benchmarking]
+default-features = false
+git = 'https://github.com/peaqnetwork/substrate'
+optional = true
+branch = 'peaq-polkadot-v0.9.29'
+
+[dependencies.frame-support]
+default-features = false
+git = 'https://github.com/peaqnetwork/substrate'
+branch = 'peaq-polkadot-v0.9.29'
+
+[dependencies.frame-system]
+default-features = false
+git = 'https://github.com/peaqnetwork/substrate'
+branch = 'peaq-polkadot-v0.9.29'
+
+[dependencies.scale-info]
+default-features = false
+features = ['derive']
+version = '2.0.1'
+
+[dependencies.sp-runtime]
+default-features = false
+git = 'https://github.com/peaqnetwork/substrate'
+branch = 'peaq-polkadot-v0.9.29'
+
+[dependencies.sp-core]
+default-features = false
+git = 'https://github.com/peaqnetwork/substrate'
+branch = 'peaq-polkadot-v0.9.29'
+
+[dependencies.sp-io]
+default-features = false
+git = 'https://github.com/peaqnetwork/substrate'
+branch = 'peaq-polkadot-v0.9.29'
+
+[dependencies.sp-std]
+default-features = false
+git = 'https://github.com/peaqnetwork/substrate'
+branch = 'peaq-polkadot-v0.9.29'  # or the latest monthly
+
+[dependencies.pallet-timestamp]
+default-features = false
+git = 'https://github.com/peaqnetwork/substrate'
+branch = 'peaq-polkadot-v0.9.29'  # or the latest monthly
+
+[dependencies.serde]
```

Comment:
I think no, because the only default feature in serde is using std, and we cannot get rid of std in serde. Doc:
https://serde.rs/feature-flags.html

In substrate / frame_support they also don't have `default-features = false`, but they have `optional=true`, but if I am honest, I don't understand this flag, why should a dependency be optional?!
[frame_support: Cargo.toml](https://github.com/peaqnetwork/substrate/blob/95067ccc3a7683768f8a10036785cde8d5f8a098/frame/support/Cargo.toml#L16)

Reply:
Oh, yep, you are right. Thanks for the explanation.